src/views/imgur/imgur.js

```diff
@@ -0,0 +1,72 @@
+const express = require('express');
+const fileUploadMiddleware = require('express-fileupload');
+const { ImgurClient } = require('imgur');
+const User = require('../../models/User');
+
+const router = express.Router();
+const imgurClient = new ImgurClient({
+  clientId: process.env.IMGUR_CLIENT_ID,
+  clientSecret: process.env.IMGUR_CLIENT_SECRET,
+  refreshToken: process.env.IMGUR_REFRESH_TOKEN,
+});
+
+router.post(
+  '/:userId/uploadDB',
+  fileUploadMiddleware({ limits: { fileSize: 10000000 }, abortOnLimit: true }),
+  async (req, res) => {
```

Comment:
This is fine for the initial work here; but for production I would expect this code to follow our current convention in this repo; where the code inside the async request handler function (line 16-69) is in a controller file, so that the controller can be more easily unit tested separate from the routes definition+middleware.

Reply:
Refactoring done , commit b7a89e1e90807bfff900463307f888d0f4ce4348